musicbrainz-dockerfile/Dockerfile

```diff
@@ -64,13 +62,7 @@ RUN eval $( perl -Mlocal::lib) && cpanm --notest \
         TURNSTEP/DBD-Pg-3.10.3.tar.gz
 
 ADD DBDefs.pm /musicbrainz-server/lib/
-ADD scripts/start.sh /start.sh
-ADD scripts/start_mb_renderer.pl /start_mb_renderer.pl
-ADD scripts/crons.conf /crons.conf
-ADD scripts/replication.sh /replication.sh
-ADD scripts/createdb.sh /createdb.sh
-ADD scripts/recreatedb.sh /recreatedb.sh
-ADD scripts/set-token.sh /set-token.sh
+ADD scripts/* /
```

Comment:
Same change as https://github.com/metabrainz/musicbrainz-docker/commit/03f749abf47aff36234e29d60ec1a2f43ffc1a90 available in the development branch [`mbvm-38-dev`](https://github.com/metabrainz/musicbrainz-docker/tree/mbvm-38-dev) already.